docs/node/migrations/zero-downtime-migrations/migration-proposal.mdx

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 1
+sidebar_label: "Migration Proposal"
+id: migration-proposal
+title: Migration Proposal
+description: "Migration Proposal"
+slug: /node/migrations/zero-downtime-migrations/migration-proposal
+---
+
+Migration Declaration is a network governance proposal to coordinate the network migration process. The migration proposal includes the activation period, duration, and the new chain ID that the network will migrate to.
+
+```go
```

Comment:
Why are you showing the Go code if the proposal process is done with `kwil-admin` ?